EIPS/eip-config.md

```diff
@@ -0,0 +1,303 @@
+
+### Serving Data Not Specified in genesis.json
+
+Some reported values are specification-level constants, which many clients do not include in their configuration files. However, certain EIP constants (e.g., the deposit contract) have become variables in testnets, necessitating their inclusion.
+
+### JSON as Config Format
+
+JSON was chosen for its ubiquity, machine and human readability, and the existence of a standardized canonical form via [RFC-8785](https://www.rfc-editor.org/rfc/rfc8785). YAML lacks a standard canonical form. No Ethereum software uses XML for configuration, and adopting it would increase every client’s library size. An invented format is possible but would require definition and standardization within this specification.
+
+### CRC-32 as Hash Format
+
+The reasons for using CRC-32 instead of a cryptographic hash are the same as those in [EIP-2124](./eip-2124#why-use-ieee-crc32-as-the-checksum-instead-of-keccak256), incorporated by reference. In brief, nodes can lie, the 4-byte reduction is for convenience rather than security, and CRC-32 is widely implemented.
+
+## Backwards Compatibility
+
+This EIP does not alter previous behavior. Configurations prior to Cancun are non-standard, and clients sharing pre-Cancun configurations will produce non-standard results.
+
+Clients supporting pre-Cancun forks MAY return partial or non-standard configurations but SHOULD strive to follow the spirit of the fields specified for Cancun configurations. Full compliance is REQUIRED only for Cancun and later forks.
+
+## Test Cases
+
+### Sample Configs
+
+Holesky Prague Config
+```JSON
+{
+  "activationTime": 1740434112,
+  "blobSchedule": {
+    "baseFeeUpdateFraction": 5007716,
+    "max": 9,
+    "target": 6
+  },
+  "chainId": "0x4268",
+  "precompiles": {
+    "0x0000000000000000000000000000000000000001": "ECREC",
+    "0x0000000000000000000000000000000000000002": "SHA256",
+    "0x0000000000000000000000000000000000000003": "RIPEMD160",
+    "0x0000000000000000000000000000000000000004": "ID",
+    "0x0000000000000000000000000000000000000005": "MODEXP",
+    "0x0000000000000000000000000000000000000006": "BN256_ADD",
+    "0x0000000000000000000000000000000000000007": "BN256_MUL",
+    "0x0000000000000000000000000000000000000008": "BN256_PAIRING",
+    "0x0000000000000000000000000000000000000009": "BLAKE2F",
+    "0x000000000000000000000000000000000000000a": "KZG_POINT_EVALUATION",
+    "0x000000000000000000000000000000000000000b": "BLS12_G1ADD",
+    "0x000000000000000000000000000000000000000c": "BLS12_G1MULTIEXP",
+    "0x000000000000000000000000000000000000000d": "BLS12_G2ADD",
+    "0x000000000000000000000000000000000000000e": "BLS12_G2MULTIEXP",
+    "0x000000000000000000000000000000000000000f": "BLS12_PAIRING",
+    "0x0000000000000000000000000000000000000010": "BLS12_MAP_FIELD_TO_CURVE",
+    "0x0000000000000000000000000000000000000011": "BLS12_MAP_FIELD_TO_CURVE"
```

Comment:
Ah another one 0x10 and 0x11 are the same precompile :thinking:

Reply:
Fixed.

Also, some other BLS names did not align with EIP names, updated those.